joomla/joomla-cms
# Joomla 3.8.0: Disabled com_fields displayed in admin menu

Pull Request for Issue #18046 .

### Summary of Changes
Added check to JAdminMenuCSS for disabled or not installed components.

### Testing Instructions
Disable com_fields, and see the menu. Links to fields components are present, leading to 404 page.
Apply this patch, and see the menu again.
